src/pages/home/report/ReportActionCompose.js

```diff
@@ -405,14 +401,8 @@ class ReportActionCompose extends React.Component {
         const isComposeDisabled = this.props.isDrawerOpen && this.props.isSmallScreenWidth;
         const isBlockedFromConcierge = ReportUtils.chatIncludesConcierge(this.props.report) && User.isBlockedFromConcierge(this.props.blockedFromConcierge);
         const inputPlaceholder = this.getInputPlaceholder();
-        const isArchivedChatRoom = ReportUtils.isArchivedRoom(this.props.report);
```

Comment:
I think you'll need to keep this for the emojiPickerButton below

Reply:
Actually, we don't need that because `ReportActionCompose` will no longer be rendered at all for archived chats.